<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>495</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
    <addaction name="actionSaveASCII"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="action_Find"/>
    <addaction name="action_clean"/>
    <addaction name="action_Option"/>
   </widget>
   <widget class="QMenu" name="menu_Device">
    <property name="title">
     <string>&amp;Device</string>
    </property>
    <addaction name="action_Connect"/>
    <addaction name="action_Disconnect"/>
    <addaction name="action_SerialConfig"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menu_Script">
    <property name="title">
     <string>&amp;Script</string>
    </property>
    <widget class="QMenu" name="menufunFile">
     <property name="title">
      <string>funFile</string>
     </property>
     <addaction name="action_fun"/>
    </widget>
    <addaction name="action_tool"/>
    <addaction name="action_selectFile"/>
    <addaction name="menufunFile"/>
    <addaction name="action_runScript"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Device"/>
   <addaction name="menu_Script"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_Undo"/>
   <addaction name="action_Redo"/>
   <addaction name="action_Find"/>
   <addaction name="action_clean"/>
   <addaction name="action_Option"/>
   <addaction name="separator"/>
   <addaction name="action_Connect"/>
   <addaction name="action_Disconnect"/>
   <addaction name="action_SerialConfig"/>
   <addaction name="separator"/>
   <addaction name="action_selectFile"/>
   <addaction name="action_runScript"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open an existing file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save the document to disk</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="text">
    <string>S&amp;aveAs</string>
   </property>
   <property name="toolTip">
    <string>Save the document under a new name</string>
   </property>
  </action>
  <action name="actionSaveASCII">
   <property name="text">
    <string>SaveASCII</string>
   </property>
   <property name="toolTip">
    <string>Save document in ASCII format</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit the application</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/find.png</normaloff>:/images/find.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Find</string>
   </property>
   <property name="toolTip">
    <string>Show the Dialog for finding and replacing</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Option">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Options</string>
   </property>
   <property name="toolTip">
    <string>Show the Dialog to select applications options</string>
   </property>
  </action>
  <action name="action_Connect">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to serial port</string>
   </property>
  </action>
  <action name="action_Disconnect">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from serial port</string>
   </property>
  </action>
  <action name="action_SerialConfig">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;SerialConfig</string>
   </property>
   <property name="toolTip">
    <string>Configure serial port</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About program</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="action_tool">
   <property name="text">
    <string>&amp;tool</string>
   </property>
  </action>
  <action name="action_selectFile">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;selectFile</string>
   </property>
   <property name="toolTip">
    <string>select script File</string>
   </property>
  </action>
  <action name="action_runScript">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/run.ico</normaloff>:/images/run.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;runScript</string>
   </property>
   <property name="toolTip">
    <string>run Script</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_fun">
   <property name="text">
    <string>./Fun.js</string>
   </property>
   <property name="toolTip">
    <string>select function lib file</string>
   </property>
  </action>
  <action name="action_clean">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>&amp;clean</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
